luigi/scheduler.py

```diff
@@ -170,7 +170,8 @@ def _get_default(x, default):
 class Task(object):
 
     def __init__(self, task_id, status, deps, resources=None, priority=0, family='', module=None,
-                 params=None, disable_failures=None, disable_window=None, disable_hard_timeout=None):
+                 params=None, disable_failures=None, disable_window=None, disable_hard_timeout=None,
+                 runnable=False):
```

Comment:
The line is at https://github.com/spotify/luigi/pull/1136/files#diff-ec5c8f6c192cef5a08a5e22aa6273604R643